packages/remark-parse/types/test.ts

```diff
@@ -1,12 +1,12 @@
 import unified = require('unified')
-import * as Unist from 'unist'
 import remarkParse = require('remark-parse')
 
 const parseOptions = {
   gfm: true,
   pedantic: true
 }
 
+unified().use(remarkParse)
```

Comment:
This test is failing currently,

Initially I thought that the extra `Parser` property on the `Attacher` might be the issue
```ts
interface Parse extends Attacher<[Partial<RemarkParseOptions>]> {
  Parser: Parser
}
```

but the tests in https://github.com/unifiedjs/unified/pull/67 passing seem to indicate otherwise.

Other ideas? :thought_balloon:

Reply:
With some more testing it looks like
```ts
// this passes
interface Parse extends Attacher {
  Parser: Parser
}

// this fails
interface Parse extends Attacher<[Partial<RemarkParseOptions>]> {
  Parser: Parser
}

// this fails
interface Parse extends Attacher<[Partial<RemarkParseOptions>]> {}
```

extending a type with a generic is causing the issue

Reply:
Found it, the settings needed to be made optional

```ts
// this passes
interface Parse extends Attacher<[Partial<RemarkParseOptions>?]> {
  Parser: Parser
}
```